dind-agent/Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM docker:1.9-dind
-RUN apk --update add openjdk7-jre unzip git python python3 bash jq openssh-client
+FROM docker:1.10-dind
+# include dependencies needed by the jenkins agent itself
```

Comment:
@rji - which of these dependencies is a strict necessity for the Jenkins slave and which aren't?

I think `openjdk8-jre`, `git` `bash` and `openssh-client`  are the essential ones - is that correct?